### PR TITLE
Create Sapphire.moe.xml

### PR DESCRIPTION
**Pull request by koops76**

### CODE REVIEW

File: src/chrome/content/rules/Sapphire.moe.xml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<!--
+	Mismatched:
+		- status (*.statuscake.com)
```
Comment (J0WI) [inline src/chrome/content/rules/Sapphire.moe.xml]:
```
pomfstatus.sapph.io
status.sapph.io

File: src/chrome/content/rules/Sapphire.moe.xml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+-->
+
+<ruleset name="Sapphire.moe">
+	<target host="sapphire.moe" />
+	<target host="www.sapphire.moe" />
+	<target host="accounts.sapphire.moe" />
+	<target host="blog.sapphire.moe" />
+	<target host="sapph.io" />
```
Comment (J0WI) [inline src/chrome/content/rules/Sapphire.moe.xml]:
```
www.sapph.io

Comment (koops76) [inline src/chrome/content/rules/Sapphire.moe.xml]: Doesn't resolve.

File: src/chrome/content/rules/Sapphire.moe.xml (hunk 1 above)
Comment (koops76) [inline src/chrome/content/rules/Sapphire.moe.xml]: Doesn't resolve.

Comment (koops76) [inline src/chrome/content/rules/Sapphire.moe.xml]: Mismatched: `reports.statuscake.com`.

File: src/chrome/content/rules/Sapphire.moe.xml (hunk 2 above)
Comment (koops76) [inline src/chrome/content/rules/Sapphire.moe.xml]: Done.

**Pull request merged**